### DISCUSSION TITLE
Example of Custom Arguments

### DISCUSSION THREAD

**Discussion by Dewdul** [Tichau/FileConverter#469] [Q&A]
I'm sorry if I am dumb D: I am looking for a way to force 10 FPS for converting from GIF to MP4, but am having trouble figuring out how to use Custom Arguments. I realize that it uses ffmpeg syntax, but I've tried various combinations of this and am not sure. The Help just takes me to the ffmpeg documentation page, but if I just use a straight example from there, it does not work. I assume I need to use the (p)(f) for the file path and name, but have tried adding this with no success. Any help or examples would be greatly appreciated! Thank you!!!! :)

**Comment by ctrlalt-666** [comment]
have you figured it out? I also dont know how to use the custom args

**Reply by Dewdul** [reply]
I feel like I did? I can't check right now, but I'll get back to you tonight

**Reply by ctrlalt-666** [reply]
Sounds good! I'm trying to batch convert mp4 files to mp3 with covers and start delays if that's possible. If not I'll try to use Mediahuman audio converter.
